src/inmanta/server/agentmanager.py

```diff
@@ -1232,19 +1232,14 @@ async def __do_start_agent(
         out: str = os.path.join(self._server_storage["logs"], "agent-%s.out" % env.id)
         err: str = os.path.join(self._server_storage["logs"], "agent-%s.err" % env.id)
 
-        agent_log = os.path.join(self._server_storage["logs"], "agent-%s.log" % env.id)
-
         proc: subprocess.Process = await self._fork_inmanta(
             [
                 "--log-file-level",
                 "DEBUG",
-                "--timed-logs",
                 "--config",
                 config_path,
                 "--config-dir",
                 Config._config_dir if Config._config_dir is not None else "",
-                "--log-file",
-                agent_log,
```

Comment:
This is now part of the log config! (I'm a bit in doubt, but it simplifies things)